docs/configuration.md

```diff
@@ -116,7 +116,7 @@ The following subsections cover all CLI flags which promscale supports. You can
 | metrics.rules.alert.for-grace-period | duration | 10 minutes | Minimum duration between alert and restored "for" state. This is maintained only for alerts with configured "for" time greater than grace period. |
 | metrics.rules.alert.for-outage-tolerance | duration | 1 hour | Max time to tolerate Promscale outage for restoring "for" state of alert. |
 | metrics.rules.alert.resend-delay | duration | 1 minute | Minimum amount of time to wait before resending an alert to Alertmanager. |
-| metrics.rules.prometheus-config | string | "" | Address of the Prometheus configuration. This is used to extract the alertmanager configuration and the addresses of rules files. Note: If this is empty or `rule_files` is empty, Promscale rule-manager will not start. If `alertmanager_config` is empty, alerting will not be initialized. |
+| metrics.rules.config-file | string | "" |  Path to configuration file in Prometheus-format, containing rule_files and optional `alerting`, `global` fields. For more details, see https://prometheus.io/docs/prometheus/latest/configuration/configuration/. Note: If this is flag or `rule_files` is empty, Promscale rule-manager will not start. If `alertmanagers` is empty, alerting will not be initialized. |
```

Comment:
The only comment I have is that this should point to our documentation on alerting and alerting configuration. I know that that documentation doesn't exist yet, so we can do this in a future change.